docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -41,8 +41,7 @@ There are different distributions available:
 
 * {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
 * production: `docker.elastic.co/package-registry/distribution:production` - Packages available in the production registry (https://epr.elastic.co).
-* staging: `docker.elastic.co/package-registry/distribution:staging` - Packages available in the staging registry (https://epr-staging.elastic.co). These packages may be pending of validation.
-* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - Packages under development.
+* lite: `docker.elastic.co/package-registry/distribution:lite` - Small subset of packages available in the production registry (https://epr.elastic.co).
```

Comment:
Do we have versioned "lite" distributions?

Reply:
I was thinking to also version that image to be offered along with the production one.
Something like: `docker.elastic.co/package-registry/distribution:lite-v8.x.y`

Related PR https://github.com/elastic/package-registry/pull/900